components/code_block/code_block.tsx

```diff
@@ -59,16 +66,54 @@ const CodeBlock: React.FC<Props> = ({code, language, searchedContent}: Props) =>
         htmlContent = `${searchedContent} ${content}`;
     }
 
+    const copyText = () => {
+        copyToClipboard(code);
+    };
+
+    const copyMarkdown = useCallback(() => {
+        const markdown = `~~~${language}\n${code}\n~~~`;
+        copyToClipboard(markdown);
+    }, [language, code]);
+
+    const contextMenu = (
+        <RootPortal>
+            <ContextMenu
+                className='post-code__context-menu'
+                id={`copy-code-block-context-menu-${id}`}
+            >
+                <MenuItem onClick={copyText}>
+                    <FormattedMessage
+                        id='copy.message'
+                        defaultMessage='Copy'
```

Comment:
These strings are detected automatically by the translation tools, but that CI job is failing right now because these strings are missing from `i18n/en.json`. You can either add them manually to that file or you can run `make i18n-extract` yourself to update the files.

Just make sure after running that that it picked up the correct English strings based on the `defaultMessage`.